freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/WorkloadCredential.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.freeipa.service.freeipa.user.model;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ActorKerberosKey;
+
+import java.util.List;
+
+public class WorkloadCredential {
+
+    private String hashedPassword;
+
+    private List<ActorKerberosKey> keys;
+
+    public WorkloadCredential(String hashedPassword, List<ActorKerberosKey> keys) {
+        this.hashedPassword = hashedPassword;
+        this.keys = keys;
```

Comment:
nit: can make both members final, and make an ImmutableList.copyOf(keys) here to be defensive and make sure the class is truly immutable